src/gfx_type.h

```diff
@@ -161,17 +160,63 @@ struct DrawPixelInfo {
 	ZoomLevel zoom;
 };
 
-/** Structure to access the alpha, red, green, and blue channels from a 32 bit number. */
-union Colour {
+#if defined(__EMSCRIPTEN__)
+
+/** Packed colour union to access the alpha, red, green, and blue channels from a 32 bit number for Empscripten build. */
```

Comment:
```suggestion
/** Packed colour union to access the alpha, red, green, and blue channels from a 32 bit number for Emscripten build. */
```

Would it be nicer if we just define all three, and at the end have an `#ifdef` to only do the `using Colour = `? It would definitely make the scope of the `#ifdef` a lot smaller.